__tests__/app.test.js

```diff
@@ -0,0 +1,34 @@
+const request = require("supertest");
+const db = require("../db/connection");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index");
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe("GET /api/topics", () => {
+  it("200: responds with an array of all the topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics } = body;
+
+        expect(topics.length).toBe(3);
+        topics.forEach((topic) => {
+          expect(typeof topic.slug).toBe("string");
+          expect(typeof topic.description).toBe("string");
+        });
+      });
+  });
+
+  it("404: responds with a 404 error when request an invalid endpoint", () => {
```

Comment:
This test is fine, but should it be within this describe block? Is it relevant to only the api/topics endpoint, or does it apply to the whole project?